rbi/stdlib/csv.rbi

```diff
@@ -1168,7 +1188,7 @@ class CSV::Table < Object
   include Enumerable
 
   extend T::Generic
-  Elem = type_member(fixed: CSV::Row)
+  Elem = type_member(:out)
```

Comment:
I had to make this change to revert https://github.com/sorbet/sorbet/pull/3156

cc @Morriar